test/compilable/commontype.d

```diff
@@ -196,7 +196,8 @@ static assert(is( X!( C***, B*** ) == const(B**)* )); // `B***`
 
 static assert(is( X!( C*, I* ) == I* ));
 static assert(is( X!( I*, C* ) == I* ));
-static assert(Error!( C**, I** ));
+//static assert(Error!( C**, I** ));
+static assert(is( X!( C**, I** ) == const(I*)* ));
```

Comment:
Is thi supposed to be `const`?

Reply:
Unsure about this, but I think so, as this is already true at L313
`static assert(is( X!( C*[4], const(B*)[4] ) == const(B*)[] )); // !?`